Tasks/PublishPackageTask.cs

```diff
@@ -58,11 +58,14 @@ public override async Task RunAsync(BuildContext context)
         projectData = projectData.Replace("{LicencePath}", context.PackContext.LicensePath);
 
         if (context.PackContext.LicensePath.EndsWith(".txt"))
-            projectData = projectData.Replace("{LicenceName}", "LICENSE.txt");
+            projectData = projectData.Replace("{LicenceName}", "LICENSE.txt").Replace ("{LicencePackagePath}", "LICENSE.txt");
         else if (context.PackContext.LicensePath.EndsWith(".md"))
-            projectData = projectData.Replace("{LicenceName}", "LICENSE.md");
+            projectData = projectData.Replace("{LicenceName}", "LICENSE.md").Replace ("{LicencePackagePath}", "LICENSE.md");
+        else if (context.PackContext.LicensePath.EndsWith ("LICENSE"))
+            // https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030#issue
+            projectData = projectData.Replace("{LicenceName}", "LICENSE").Replace ("{LicencePackagePath}", "");
         else
-            projectData = projectData.Replace("{LicenceName}", "LICENSE");
+            projectData = projectData.Replace("{LicenceName}", "LICENSE").Replace ("{LicencePackagePath}", "LICENSE");
```

Comment:
Shouldn't `LicencePackagePath` be set to "" here as well?

Reply:
`else if (context.PackContext.LicensePath.EndsWith ("LICENSE"))` becomes redundant if true so it can be removed.